addons/stock/views/stock_quant_views.xml

```diff
@@ -77,51 +77,14 @@
         </field>
     </record>
 
-    <record model="ir.ui.view" id="view_stock_quant_form">
```

Comment:
Should keep it due to ORM when you use create in an empty editable list view with a groupby